### PR TITLE
Add data and scripts for 2017-10-05 monthly update

### PR DESCRIPTION
**Pull request by nacnudus**
The entries to be appended to the register are in the file
lists/append-entries-2017-10-05.tsv

### CODE REVIEW

File: lists/new-entries.tsv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+government-organisation	hostname	organisation	start-date	end-date
+17597	denhambucks-pc		2017-09-19	""
```
Comment (Jamie-Atkinson) [inline lists/new-entries.tsv:2]: should not have ""

File: lists/append-entries-2017-10-05.tsv [hunk 2]
```diff
@@ -0,0 +1,15 @@
+government-domain	hostname	organisation	start-date	end-date
+7448	volcomgrantsni	government-organisation:OT1176	2003-02-12	NA
```
Comment (Jamie-Atkinson) [inline lists/append-entries-2017-10-05.tsv]: should not have NA on multiple lines in this file

File: lists/append-entries-2017-10-05.tsv [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+6539	proni	government-organisation:OT1176	2002-10-09	NA
+4762	dsdni	government-organisation:OT1175	2005-04-18	NA
+4637	dcalni	government-organisation:OT1174	2006-08-18	NA
+15597	boundarycommissionengland	local-authority-eng:BMH	2014-11-05	NA
```
Comment (Jamie-Atkinson) [inline lists/append-entries-2017-10-05.tsv]: this should be linked to government-organisation:PB337

File: lists/map-correction.tsv [hunk 4]
```diff
@@ -0,0 +1,7 @@
+government-domain	hostname	organisation	start-date	end-date
```
Comment (Jamie-Atkinson) [inline lists/map-correction.tsv:1]: all these corrections are already in the register?

**Review by Jamie-Atkinson — CHANGES_REQUESTED** [review]
Few issues/ questions duncan.

File: lists/map-correction.tsv (hunk 4 above)
Comment (nacnudus) [reply lists/map-correction.tsv:1]: The 'map-corrections.tsv' file got overwritten by the original data.  I've pushed a fix.

File: lists/new-entries.tsv (hunk 1 above)
Comment (nacnudus) [reply lists/new-entries.tsv:2]: Pushed a fix.

**Comment by nacnudus** [comment]
Here's the new PR checklist

* [ ] The file is tab-delimited
* [ ] The field names match the register metadata in [registry-data](https://github.com/openregister/registry-data)
* [ ] The rows are unique
* [ ] The [datatypes](https://datatype.register.gov.uk/records) are correct
    * [ ] string
    * [ ] text
    * [ ] datetime (ISO-8601)
    * [ ] url
    * [ ] curie
* [ ] New entries are at the bottom (unless this is for a new load / reload
* [ ] NA values are represented by nothing, not by "NA" or ""
* [ ] The source data file is in the repo, or is described in a commit message
* [ ] The new data matches the source data
* [ ] Any new curies exist in the registers they link to

**Review by Jamie-Atkinson — APPROVED** [review]

**Pull request merged**